net/unbound/files/unbound.sh

```diff
@@ -1047,9 +1026,28 @@ unbound_uci() {
 
 ##############################################################################
 
+unbound_ismain() # true=0, false=1.
+{
+  if [ "$UNBOUND_N_RX_PORT" != "53" ] ; then
```

Comment:
Please use the previous "is dnsmasq enable". It needs to be two parts. The previous test and port 53 conflict check. It is possible that both dnsmasq and Unbound UCI have 53. The two are enabled/disabled to run back to back testing. This how I test (Unbound+odhcpd) vs (dnsmasq-full alone) for feature commonality.